app/views/discussions/terms.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="mu-user-header">
+  <h1><%= t :forum_terms %></h1>
+</div>
+
+<div>
+  <% @forum_terms.each do |term| %>
```

Comment:
Title is not should on purpose, right?

Reply:
That's right, it seemed redundant

Reply:
That why they don't have all the summary logic neither. It's just one big header